index.html

```diff
@@ -53,26 +60,64 @@
                 </div>
                 <div class="resume-item">
                     <div class="dates">
-                        June 17' - September '17'
+                        Summer '16
                     </div>
                     <div class="title">
                         PM - Eriks West
                     </div>
                     <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Mauris id sagittis
                         risus, at dictum nunc. Proin viverra enim vel diam molestie, at scelerisque odio sagittis. </p>
                 </div>
+                <div class="resume-item">
+                    <div class="dates">
+                        Summer '15
+                    </div>
+                    <div class="title">
+                        Entry Automation - Local Nonprofit
+                    </div>
+                    <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Mauris id sagittis
+                        risus, at dictum nunc. Proin viverra enim vel diam molestie, at scelerisque odio sagittis. </p>
+                </div>
             </div>
             <div class="resume-items">
+                <h1>Projects</h1>
                 <div class="resume-item">
                     <div class="dates">
-                        May '19 - April '20'
+                        July '20
+                    </div>
+                    <div class="title">
+                        <a href="https://Streampad.io">StreamPad.io</a>
```

Comment:
For URLs that go out of your current website, I'd recommend opening them in new tabs. You can do this by adding the `target` attribute with a value of `"_blank"`, but there's this nieche security vulnerability you'll wanna be aware of. No need to really understand it right now, but if you're curious: https://web.dev/external-anchors-use-rel-noopener/

```suggestion
                        <a href="https://Streampad.io" target="_blank" rel="noopener noreferrer">StreamPad.io</a>
```